testing/jormungandr-integration-tests/src/jormungandr/rest/v1/utils/settings.rs

```diff
@@ -0,0 +1,137 @@
+    let reward_param =
+        match blockchain_configuration.reward_parameters{
+            None => return None,
+            Some(r) => r
+        };
+
+    let reward_drawing =
+         match blockchain_configuration.reward_constraints.reward_drawing_limit_max{
+            None => Limit::None,
+            Some(r) => Limit::ByStakeAbsolute(r.into())
+        };
+    //TODO use map?
+    let pool_participation =
+        match blockchain_configuration.reward_constraints.pool_participation_capping
+         {
+            None => None,
+            Some(p) => Some((p.min, p.max))
+        };
+
+    match reward_param {
+        RewardParams::Linear {
+            constant,
+            ratio,
+            epoch_start,
+            epoch_rate,
+        } => Some(Parameters {
+            initial_value: constant,
+            compounding_ratio: ratio.into(),
+            compounding_type: CompoundingType::Linear,
+            epoch_rate,
+            epoch_start,
+            reward_drawing_limit_max: reward_drawing,
+            pool_participation_capping: pool_participation,
+        }),
+        RewardParams::Halving {
+            constant,
+            ratio,
+            epoch_start,
+            epoch_rate,
+        } => Some(Parameters {
+            initial_value: constant,
+            compounding_ratio: ratio.into(),
+            compounding_type: CompoundingType::Halvening,
+            epoch_rate,
+            epoch_start,
+            reward_drawing_limit_max: reward_drawing,
+            pool_participation_capping: pool_participation,
+        }),
+    }
+}
+
+fn get_settings_from_block0_configuration(block0_configuration: &Block0Configuration) -> SettingsDto{
+    let blockchain_configuration = &block0_configuration.blockchain_configuration;
+    SettingsDto {
+        block0_hash: block0_configuration.to_block().header().id().to_string(),
+        block0_time: blockchain_configuration.block0_date.into(),
+        //TODO use map?
+        curr_slot_start_time: Some(SystemTime::from(blockchain_configuration.block0_date)),
+        consensus_version: blockchain_configuration.block0_consensus.to_string(),
+        fees: blockchain_configuration.linear_fees,
+        block_content_max_size: blockchain_configuration.block_content_max_size.into(),
+        epoch_stability_depth: blockchain_configuration.epoch_stability_depth.into(),
+        slot_duration: u8::from(blockchain_configuration.slot_duration).into(),
+        slots_per_epoch: blockchain_configuration.slots_per_epoch.into(),
+        //TODO handle the unwarp
+        treasury_tax: blockchain_configuration.treasury_parameters.unwrap().into(),
+        reward_params: get_parameters_from_blockchain_config(blockchain_configuration).unwrap(),
+        discrimination: blockchain_configuration.discrimination,
+        tx_max_expiry_epochs: blockchain_configuration.tx_max_expiry_epochs.unwrap(),
+    }
+
+}
+
+#[rstest]
```

Comment:
rstest is useful when you have some setup before. in this case, `#[test]` will do